ipaplatform/base/services.py

```diff
@@ -58,7 +58,7 @@
     'named', 'ods_enforcerd', 'ods_signerd', 'gssproxy',
     'nfs-utils', 'sssd', 'NetworkManager', 'ipa-custodia',
     'ipa-dnskeysyncd', 'ipa-otpd', 'ipa-ods-exporter',
-    'systemd-resolved',
+    'systemd-resolved', 'systemd-timesyncd',
```

Comment:
I don't think this is needed here since it isn't a service IPA needs to know, just one that may get shut down in favor of another time daemon.

Reply:
fixed